viadot/flows/sap_rfc_to_adls.py

```diff
@@ -0,0 +1,118 @@
+import pandas as pd
+from typing import Any, Dict, List, Literal
+from prefect import Flow, task, unmapped
+
+from viadot.tasks import SAPRFCToDF
+from viadot.tasks import AzureDataLakeUpload
+from viadot.task_utils import df_to_csv
+
+download_sap_task = SAPRFCToDF()
+file_to_adls_task = AzureDataLakeUpload()
+
+
+@task
+def concat_dfs(dfs: List[pd.DataFrame]):
+    """
+    Task to combine list of data frames into one
+
+    Args:
+        dfs (List[pd.DataFrame]): List of dataframes to concat.
+    Returns:
+        full_df (pd.DataFrame()): Pandas dataframe containing all columns from dataframes from list.
+    """
+    full_df = pd.DataFrame()
+    for df in dfs:
+        full_df = pd.concat([full_df, df], axis=1)
+    return full_df
+
+
+class SAPRFCToADLS(Flow):
+    def __init__(
+        self,
+        name: str,
+        query_list: List[str],
+        sep: str = None,
+        func: str = "BBP_RFC_READ_TABLE",
+        sap_credentials: dict = None,
+        local_file_path: str = None,
+        file_sep: str = "\t",
+        if_exists: Literal["append", "replace", "skip"] = "replace",
+        adls_path: str = None,
+        overwrite: bool = False,
+        sp_credentials_secret: str = None,
```

Comment:
This should specify `adls` since we will eventually use two credential secrets, one for SAP and one for ADLS. So I would call it eg. `adls_sp_credentials_secret`, and then later on we will have `sap_credentials_secret`.